app/app_xdg.go

```diff
@@ -116,15 +124,11 @@ func rootConfigDir() string {
 }
 
 func watchTheme() {
-	go func() {
-		// with portal this may not be immediate, so we update a cache instead
```

Comment:
This is the comment that explained why it was in a goroutine before the move to `init`